AutomatedTesting/Gem/PythonTests/scripting/NewScriptEventButton_HappyPath_ContainsSCCategory.py

```diff
@@ -53,18 +49,19 @@ class TestAssetEditor_NewScriptEvent:
     async def run_test(self):
         # 1) Open Script Canvas window (Tools > Script Canvas)
         general.idle_enable(True)
-        general.open_pane("Script Canvas")
-        helper.wait_for_condition(lambda: general.is_pane_visible("Script Canvas"), 5.0)
+        general.open_pane(SCRIPT_CANVAS_UI)
+        helper.wait_for_condition(lambda: general.is_pane_visible(SCRIPT_CANVAS_UI), WAIT_TIME_3)
 
         # 2) Close any existing AssetEditor window
-        general.close_pane("Asset Editor")
-        helper.wait_for_condition(lambda: not general.is_pane_visible("Asset Editor"), 5.0)
+        general.close_pane(ASSET_EDITOR_UI)
+        helper.wait_for_condition(lambda: not general.is_pane_visible(ASSET_EDITOR_UI), WAIT_TIME_3)
```

Comment:
Is there an issue with reducing the wait time here? What's been identified as the root cause of the test failure here? Are these standard workflows that we're using and correcting bad test consistency?